packages/pill/src/__tests__/Pill.snapshot.test.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import Pill from '../Pill';
+
+const defaultChild = '$100 off';
+
+describe('Pill Snapshots tests', () => {
+  it('should match snapshot', () => {
+    expect(render(<Pill>{defaultChild}</Pill>)).toMatchSnapshot();
+  });
+
+  it('should render text', () => {
+    expect(render(<Pill>{defaultChild}</Pill>)).toMatchSnapshot();
+  });
```

Comment:
what is the difference between these tests cases ? 
and where is the test with icon prop ?

Reply:
resolved